pkg/option/config.go

```diff
@@ -1536,6 +1545,12 @@ type DaemonConfig struct {
 	// EnableNat46X64Gateway is true when L3 based NAT46 and NAT64 translation is enabled
 	EnableNat46X64Gateway bool
 
+	// EnableStandaloneDNSProxy is the option to enable standalone DNS proxy
+	EnableStandaloneDNSProxy bool
```

Comment:
New variables should not be introduced to `DaemonConfig`. They should be added to a feature specific config struct registered with `cell.Config`. Though I can see that perhaps in the case of `pkg/fqdn` it might not yet be possible without a bigger refactoring.

Reply:
I need these config in cilium agent too. Is that possible with the new refactoring we have for `fqdn/service` ?

Reply:
if you define the config with `cell.Config(...)` in the "parent" [fqdn cell](https://github.com/cilium/cilium/blob/main/pkg/fqdn/cell/cell.go#L15) you should be able to inject and use the config in all  sub-cells (`fqdn-bootstrap` & `sdp-grpc-server`)

Reply:
I added to `sdp-grpc-server` because the flag is currently needed in that package only.